### DISCUSSION TITLE
Add extra references

### DISCUSSION THREAD

**Discussion by Pvmdark** [cwapi3d/cwapi3dpython#57] [Ideas]
Dear,

Is it possible to add the references below:
Element controller convert rectangular beam to circular beam
The command round machine rough part with options Positive/negative height axis and positive/negatie width axis
Checking on defective parts and option to activate them.
Option to add command surfaces to volumes

Kind regards,
Branko

**Comment by roehligas** [comment]
Volume to surfaces would also be nice... :)
